src/main/java/org/spongepowered/common/mixin/core/world/ExplosionMixin.java

```diff
@@ -146,10 +149,11 @@ public void doExplosionA() {
                                 final IBlockState iblockstate = this.world.getBlockState(blockpos);
 
                                 if (iblockstate.getMaterial() != Material.AIR) {
-                                    final float f2 = this.exploder != null
-                                               ? this.exploder.getExplosionResistance((net.minecraft.world.Explosion) (Object) this
-                                            , this.world, blockpos, iblockstate)
-                                               : iblockstate.getBlock().getExplosionResistance((Entity) null);
+                                    final float f2 = impl$resistanceCalculator.calculateResistance(
```

Comment:
`this.impl$resistanceCalculator.`...

Also, because this is an `Overwrite`, you MUST add `// Sponge start - (reason)` and `// Sponge end` comments where you make a change. It is crucial we know what the diffs are. You should also add to the javadoc on this method to explain what change you've made and why.

Reply:
oops, will get right on that.

Reply:
Think I've corrected this problem?